test/unit/Migration.t.sol

```diff
@@ -11,28 +11,26 @@ import { Proxy } from "../../src/Proxy.sol";
 
 import { MockM, MockRegistrar } from "./../utils/Mocks.sol";
 
-contract WrappedMTokenV2 {
+contract WrappedMTokenV3 {
     function foo() external pure returns (uint256) {
         return 1;
     }
 }
 
-contract WrappedMTokenMigratorV1 {
-    bytes32 private constant _IMPLEMENTATION_SLOT =
-        bytes32(0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc);
+contract WrappedMTokenMigratorV2 {
+    uint256 private constant _IMPLEMENTATION_SLOT = 0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc;
 
     address public immutable implementationV2;
```

Comment:
Shouldn't it be `implementationV3` instead of `implementationV2`?
```suggestion
    address public immutable implementationV3;
```

Reply:
The purposes of this test are to check that the current implementation and source can upgrade to a next implementation. So source is v2 and therefore the test checks if it can go to a v3.